tests/merkle-dag-tests.js

```diff
@@ -154,7 +154,22 @@ module.exports = function (repo) {
       })
     })
 
-    it('get a mdag node', (done) => {
+    it('get a mdag node from base58 encoded string', (done) => {
+      const node = new DAGNode(new Buffer('more data data data'))
+      dagService.add(node, (err) => {
+        expect(err).to.not.exist
+        var mh = node.multihash()
+        var encodedMh = bs58.encode(mh)
```

Comment:
Could you use a static string whose value is what you're expecting? This makes the test more explicit and doesn't rely on generated values to test assumptions.